axum/src/extract/multipart.rs

```diff
@@ -142,6 +142,11 @@ impl<'a> Field<'a> {
     pub async fn text(self) -> Result<String, MultipartError> {
         self.inner.text().await.map_err(MultipartError::from_multer)
     }
+    
+    // Stream a chunk of the field data.
```

Comment:
This should be a doc comment.

I think we should also include

> When the field data has been exhausted, this will return `None`.

It's in multer's docs as well.

I think we should also mention that this does the exact same as the `Stream` implementation. Otherwise that isn't clear.

Reply:
Yup make sense. This has been added along with an example